### PR TITLE
cli: Adding --remote flag on "flynn create" command

### PR DESCRIPTION
**Pull request by jvatic**
This replaces #717

### CODE REVIEW

**Comment by titanous** [comment]
Squash ad49402345b5c1c0de7fdbb43305aa6ff35dda1d into 03312134ab1b868e70e06b10d9487ffed534cf3d (and don't forget to update or remove the commit sha in 145b17ea65556b9d05507d18b60ceb6d061597d5).

File: cli/app.go [hunk 1]
```diff
@@ -36,7 +40,9 @@ If run from a git repository with a 'flynn' remote for the app, it will be
 removed.
 
 Options:
-	-y, --yes  Skip the confirmation prompt.
+	-r, --remote <remote>  Name of git remote to delete, empty string for none. [default: flynn]
+	-y, --yes              Skip the confirmation prompt.
+	--app <name>           Explicitly set name of app to delete (otherwise found automatically if --remote is not an empty string).
```
Comment (titanous) [inline cli/app.go]: There is already a global [`-a` flag](https://github.com/flynn/flynn/blob/bd2ac6ed88ad4ef4192da84ecde1545e8a5fab0b/cli/main.go#L93-L103), so `--app` should be redundant.

**Comment by jvatic** [comment]
@titanous comments addressed

**Comment by titanous** [comment]
LGTM

**Pull request merged**